### ISSUE TITLE
Conflicts with Angular 13

### ISSUE THREAD

**Issue by jakguel** [vlafranca/ngxAutocomPlace#27]
I'm trying to upgrade my project to angular 13, but I'm getting the following error every time I run ```npm install```

```js
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: Project.de@2.0.2
npm ERR! Found: @angular/common@13.2.2
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^13.2.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^10.0.0" from ngx-autocom-place@4.0.0
npm ERR! node_modules/ngx-autocom-place
npm ERR!   ngx-autocom-place@"^4.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

As it seems, this package is not supporting angular 13

**Comment by vlafranca** [comment]
Thanks for report, I will fix this soon.

**Comment by vlafranca** [comment]
I don't reproduce the issue with a fresh Angular 13 project the lib install just fine. Do you have a playground to reproduce ?

**Comment by vlafranca** [comment]
Closing this for inactivity.
A new version has been released with an update to peer dependencies 4.0.1.
